Vizzuality/half-earth-v3
# Feature/cypress automation

## Automating test suite execution on commit push
### Description
This PR sets up a github workflow aimed to run tests on each push to the repository.
Several community build actions are used in this workflow:
- [Checkout](https://github.com/actions/checkout) 
- [Setup-node](https://github.com/actions/setup-node)
- [Cypress github action](https://github.com/cypress-io/github-action)

A branch protection rule has been added to avoid `feature/*` branches to be able to be delivered without passing tests. 

### Testing instructions
Check the [github actions tab](https://github.com/Vizzuality/half-earth-v3/actions) on the repo and you'll see the workflow acting.
